src/components/AdventureDetail.jsx

```diff
@@ -140,6 +140,10 @@ function getAdventure(data) {
 		// expect there only to be a single adventure in the array
 		if (data.adventureList.items.length === 1) {
 			return data.adventureList.items[0];
+		} else {
+			return data.adventureList.items.filter(item => {
+				return item._path.startsWith("/content/dam/wknd-shared/en");
+			})[0];
```

Comment:
Could be simplified to `return data.adventureList.items.filter(item => item._path.startsWith("/content/dam/wknd-shared/en"))[0];`

Reply:
also could be find() and not filter, then it'd just return the first found item

Reply:
...but that's just a comment, unimportant

Reply:
I like the find... changed it :)